.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+---
+
+**What steps did you take and what happened:**
+[A clear and concise description of what the bug is.]
+
+
+**What did you expect to happen:**
+
+
+**Anything else you would like to add:**
+[Miscellaneous information that will assist in solving the issue.]
+
+
+**Environment:**
+
```

Comment:
probably we would also need to get the version if users runs CLI to verify?

Reply:
which version are you referring to? We're asking for the ratify version below so that would cover the controller image/CLI version?

Reply:
Users can use Ratify as CommandLine and Add-on to K8s, I feel we need to cover both?